File: node-client/worker.js
var jackrabbit = require('jackrabbit');
var rabbit = jackrabbit(process.env.RABBIT_URL);

var exchange = rabbit.fanout('broadcast');
var eventsQueue = exchange.queue({ durable: true, exclusive: true });


eventsQueue.consume(onMessage, { noAck: true });

function onMessage(data, ack) {
  console.log('received:', data);
}

File: python-client/worker.py
import os
from kombu import Connection, Queue, Exchange, common

RABBIT_URL = os.environ['RABBIT_URL']

events_exchange = Exchange('broadcast', type='fanout', durable=True)
events_queue = Queue(name=common.uuid(), exchange=events_exchange)


def process_msg(body, message):
    print('Received: {}'.format(body))
    message.ack()


with Connection(RABBIT_URL) as conn:
    with conn.Consumer(events_queue, callbacks=[process_msg]) as consumer:
        while True:
            conn.drain_events()
